branch_create.go

```diff
@@ -13,7 +13,13 @@ import (
 	"go.abhg.dev/gs/internal/text"
 )
 
+type branchCreateConfig struct {
```

Comment:
wasn't sure if you wanted this struct to be private or public, or where it should go. Happy to move it.

Reply:
Private is good for now 👍